recipes/_system_tuning.rb

```diff
@@ -26,6 +26,12 @@
   value 0
 end
 
+# Prevent Linux from using Hadoop ports as client ports (COOK-79)
+ports = node['hadoop']['sysctl']['net.ipv4.ip_local_reserved_ports'].uniq.to_s.tr(' ','').tr('[','').tr(']','')
```

Comment:
So this will overwrite any settings the user has set via other cookbooks (or outside of chef).  We should add in any attrs specified in ``node['sysctl']['params']`` to at least make it possible to play nicely with ports specified in other cookbooks on the same node.